reference/tools/tiup/manage-component.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Manage TiUP Components with TiUP Commands
+summary: Learn how to manage TiUP components using TiUP commands.
+category: tools
+---
+
+# Manage TiUP Components with TiUP Commands
+
+You can use the following TiUP commands to manage components in the TiUP ecosystem:
+
+- list: Queries the component list. By using this TiUP command, you can see all the optional components to install and all the optional versions of each component.
+- install: Installs the specific version of a component.
+- update: Updates a component to the latest version.
+- uninstall: Uninstalls a component.
+- status: Checks the status of a running component.
+- clean: Cleans up the instance on which a component is deployed.
+- help: Prints the help information. If you append another TiUP command to this command, the usage of the appended command is printed.
+
+This document introduces the common component management operations and the corresponding TiUP commands.
+
+## Query the component list
+
+You can use the `tiup list` command to query the component list. This usage of this command is as follows:
+
+- `tiup list`: checks which components can be installed.
+- `tiup list ${component}`: checks which versions of a specific component can be installed.
+
+You can also use the following two flags in the above commands:
+
+- `--installed`: checks which components or which version of a specific component has been installed locally.
+- `--refresh`: gets the latest list of components on the server and the version list of each component.
+
+Example 1: View all currently installed components.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup list --installed
+```
+
+Example 2: Get a list of the TiKV component of all installable versions from the server.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup list tikv --refresh
+```
+
+## Install components
+
+You can use the `tiup install` command to query the component list. This usage of this command is as follows:
+
+- `tiup install <component>`: installs the latest stable version of a specified component.
+- `tiup install <component>:[version]`: installs the specified version of a specified component.
+
+Example 1: Use TiUP to install the latest stable version of TiDB.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup install tidb
+```
+
+Example 2: Use TiUP to install the nightly version of TiDB.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup install tidb:nightly
+```
+
+Example 3: Use TiUP to install TiKV v3.0.6.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup install tikv:v3.0.6
+```
+
+## Upgrade components
+
+After PingCAP has released a new version of a component, you can use the `tiup update` command to upgrade this component. The usage of this command is basically the same as that of `tiup install`, except for the following flags:
```

Comment:
```suggestion
After a new version of a component is published, you can use the `tiup update` command to upgrade this component. The usage of this command is basically the same as that of `tiup install`, except for the following flags:
```
The owner of a TiUP component is not necessarily PingCAP. The TiUP ecosystem is open-sourced.